Scripts/GradientsTypeScript.swift

```diff
@@ -2,45 +2,69 @@
 
 import Foundation
 
+func getColorLiteral(hexString: String)->String{
```

Comment:
I think we can make the name more swifty, for example: `colorLiteral(for hexString: String) -> String`
Also, I think SwiftLint doesn't run the script, so don't forget the spaces: `)->String{` -> `) -> String {`